src/intl/messages.js

```diff
@@ -304,6 +304,8 @@ export const messages = {
     message: 'Are you sure you want to drop your changes?',
     description: 'Message in the modal dialog opened when a user tried to navigate off an editor page after changes have been made.',
   },
+  globalErrorBoundaryTitle: 'We are currently having some issues with VM portal.',
+  globalErrorBoundaryDescription: 'Please refresh page or log out and log back in. If the issue persists, please report a bug on GitHub: ',
```

Comment:
change to: "Please refresh the page or log out and log back in. If the issue persists, please report a bug on [link to issue tracker] github."  (missing "the" and "GitHub" should be the link)